### DISCUSSION TITLE
External Plugin for Mega Menu

### DISCUSSION THREAD

**Discussion by robertmain53** [flyntwp/flynt#391] [Q&A] [answered]
Did anybody had good experience with mega menu in FlyntWP?

I'm wondering if there is some already coded solution out there.

Thanks

**Comment by aaronmeder** [comment, marked as answer]
I would also be interested, since I might need this on an upcoming project :-)

💡 As a general approach you can replace the NavigationMain and/or NavigationBurger components with your own NavigationMegaMenu.

**Reply by aaronmeder** [reply]
@robertmain53 just on update on how I did it:

- I've added normal ACF fields to my menu items to add an image for the mega menu
- I use a custom [WP Menu Walker](https://awhitepixel.com/blog/wordpress-menu-walkers-tutorial/) to output these fields when I output my menu.

Isn't super elegant but it works while staying pretty close to the standard WP way of doing things.
